app/models/exam_template.rb

```diff
@@ -221,8 +221,20 @@ def save_pages(partial_exams)
             division.start <= page_num && page_num <= division.end
           end
         end
+        extra_pages.sort_by! { |page_num, _| page_num }
         extra_pdf = CombinePDF.new
-        extra_pdf << extra_pages.collect { |_, page| page }
+        cover_pdf = CombinePDF.new
+        start_page = 0
+        if extra_pages[0][0] == 1
+          cover_pdf << extra_pages[0][1]
+          txn.add(File.join(assignment_folder,
```

Comment:
Error-handling is a little tricky. If we only save a `COVER.pdf` file when it's there, there will be an error raised in the Assign Scans page. So maybe it's better to save an "empty PDF" (if that's even possible) so it always exists?